src/System.Management.Automation/engine/parser/SemanticChecks.cs

```diff
@@ -458,13 +458,12 @@ public override AstVisitAction VisitSwitchStatement(SwitchStatementAst switchSta
             // Parallel flag not allowed
```

Comment:
@KirkMunro, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Scripting\ForeachParallel.Tests.ps1:84 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```